### DISCUSSION TITLE
How to save to a db when using yjs with Slate?

### DISCUSSION THREAD

**Discussion by SamHatoum** [BitPhinix/slate-yjs#105] [Q&A] [answered]
Hey there

So if I want to save a doc to a db using slate, I can serialize and save that content.

But when multiple clients are collaborating at the same time, what's the best practice to save to a db?

Would love to hear about your thoughts / experience.

Thanks

Sam

**Comment by BitPhinix** [comment, marked as answer]
Hi @samhatoum,

there are numerous ways of doing so, most of them are addressed here: https://github.com/yjs/yjs/issues/170

Hope this helps

Eric

**Reply by SamHatoum** [reply]
thank you
